resources/views/admin/users.blade.php

```diff
@@ -0,0 +1,111 @@
+                            <td>Kei Sasaki</td>
+                            <td>kei@gmail.com</td>
+                            <td>20xx-xx-xx</td>
+                            <td>20xx-xx-xx</td>
+                            <td><i class="fa-solid fa-circle text-success"></i></td>
+                            <td><i class="fa-solid fa-ellipsis"></i></td>
+                        </tr>
+                        <tr class="color-2">
+                            <td>2</td>
+                            <td><i class="fa-solid fa-circle-user avatar-lg"></i></td>
+                            <td>Kei</td>
+                            <td>Kei Sasaki</td>
+                            <td>kei@gmail.com</td>
+                            <td>20xx-xx-xx</td>
+                            <td>20xx-xx-xx</td>
+                            <td><i class="fa-solid fa-circle text-success"></i></td>
+                            <td><i class="fa-solid fa-ellipsis"></i></td>
+                        </tr>
+                        <tr >
+                            <td>3</td>
+                            <td><i class="fa-solid fa-circle-user avatar-lg"></i></td>
+                            <td>Kei</td>
+                            <td>Kei Sasaki</td>
+                            <td>kei@gmail.com</td>
+                            <td>20xx-xx-xx</td>
+                            <td>20xx-xx-xx</td>
+                            <td><i class="fa-solid fa-circle text-success"></i></td>
+                            <td><i class="fa-solid fa-ellipsis"></i></td>
+                        </tr>
+                        <tr class="color-2">
+                            <td>4</td>
+                            <td><i class="fa-solid fa-circle-user avatar-lg"></i></td>
+                            <td>Kei</td>
+                            <td>Kei Sasaki</td>
+                            <td>kei@gmail.com</td>
+                            <td>20xx-xx-xx</td>
+                            <td>20xx-xx-xx</td>
+                            <td><i class="fa-solid fa-circle text-success"></i></td>
+                            <td><i class="fa-solid fa-ellipsis"></i></td>
+                        </tr>
+                        <tr>
+                            <td>5</td>
+                            <td><i class="fa-solid fa-circle-user avatar-lg"></i></td>
+                            <td>Kei</td>
+                            <td>Kei Sasaki</td>
+                            <td>kei@gmail.com</td>
+                            <td>20xx-xx-xx</td>
+                            <td>20xx-xx-xx</td>
+                            <td><i class="fa-solid fa-circle text-success"></i></td>
+                            <td><i class="fa-solid fa-ellipsis"></i></td>
+                        </tr>
+                    </tbody>
+                </table>
+            </div>
+
+        </div>
+        <hr>
+        <div class="col footer pt-3 ps-3">
+            showing 1 to 10 of 50 users
```

Comment:
is this static in the end or just putting temporary here?

Reply:
it's just temporary. I'm going to change later.